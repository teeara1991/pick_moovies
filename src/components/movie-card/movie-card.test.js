import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from '../movie-card/movie-card.jsx';

it(`MovieCard correctly renders after relaunch`, () => {
  const tree = renderer
      .create(<MovieCard
        key={3}
        title={`Fantastic Beasts`}
        logo={``}
        links={``}
        onPreview={() => {}}
        isActive={true}
        onChangeState={() =>{}}
      />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
