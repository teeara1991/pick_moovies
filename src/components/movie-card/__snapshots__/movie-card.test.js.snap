// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard correctly renders after relaunch 1`] = `
<article
  className="small-movie-card catalog__movies-card active"
  onMouseOver={[Function]}
>
  <button
    className="small-movie-card__play-btn"
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Fantastic Beasts"
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href=""
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
